fluent-bit-kinesis.go

```diff
@@ -273,6 +274,8 @@ func unpackRecords(kinesisOutput *kinesis.OutputPlugin, data unsafe.Pointer, len
 			timestamp = time.Now()
 		}
 
+		record = enricher.EnrichRecord(record, timestamp)
```

Comment:
Is it possible to make this configurable (enable/disable) as a flag? That way we don't need to rebuild the image to enable/disable it

Reply:
Added https://github.com/Canva/amazon-kinesis-streams-for-fluent-bit/pull/2/commits/048ea79d46fcdbd15ab642355c94f786990832a0